api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -113,20 +118,42 @@ public boolean isUsernameTaken(String username) {
   public User createUser(String givenName, String familyName, String username, String contactEmail) {
     String gSuiteDomain = configProvider.get().googleDirectoryService.gSuiteDomain;
     String password = randomString();
+    Alias alias = new Alias();
+    alias.setAlias(contactEmail);
     User user = new User()
       .setPrimaryEmail(username+"@"+gSuiteDomain)
       .setPassword(password)
       .setName(new UserName().setGivenName(givenName).setFamilyName(familyName))
-      .setEmails(new UserEmail().setType("custom").setAddress(contactEmail).setCustomType("contact"))
+//      .setAliases(Arrays.asList(contactEmail))
```

Comment:
This is where I'm currently struggling. I've now tried multiple ways to set the custom email field ("contact") to the user's contact email. But can NOT seem to figure it out. 
I think the alternative is to just not set this in the google account, and only use our database for that info, but not loving that approach.